src/wallet/test/fuzz/notifications.cpp

```diff
@@ -69,14 +69,13 @@ struct FuzzedWallet {
     CScript GetScriptPubKey(FuzzedDataProvider& fuzzed_data_provider)
     {
         auto type{fuzzed_data_provider.PickValueInArray(OUTPUT_TYPES)};
-        BResult<CTxDestination> op_dest;
+        util::Result<CTxDestination> op_dest{util::Error{}};
         if (fuzzed_data_provider.ConsumeBool()) {
             op_dest = wallet->GetNewDestination(type, "");
         } else {
             op_dest = wallet->GetNewChangeDestination(type);
         }
-        assert(op_dest.HasRes());
-        return GetScriptForDestination(op_dest.GetObj());
+        return GetScriptForDestination(*Assert(op_dest));
```

Comment:
Add `util/check.h` include header for `Assert`

Can drop the temporary here, if desired
```diff 
     // Add tx to wallet
     const auto& op_dest = wallet.GetNewDestination(OutputType::BECH32M, "");
     BOOST_ASSERT(op_dest);
-    const CTxDestination& dest = *op_dest;
 
     CMutableTransaction mtx;
-    mtx.vout.push_back({COIN, GetScriptForDestination(dest)});
+    mtx.vout.push_back({COIN, GetScriptForDestination(*op_dest)});
```

Reply:
Picked up in #/25616.